javascript/tbdex-pfi-exemplar/src/seed-offerings.ts

```diff
@@ -224,7 +224,7 @@ async function main() {
                     path: ['$.type[*]'],
                     filter: {
                       type: 'string',
-                      pattern: '^SanctionCredential$',
+                      pattern: '^KnownCustomerCredential$',
```

Comment:
there was a change to the spec. it no longer looks for `type`. It looks for `credentialSchema.id`:

```json
              {
                "path": [
                  "$.credentialSchema[*].id"
                ],
                "filter": {
                  "type": "string",
                  "const": "https://vc.schemas.host/kcc.schema.json"
                }
              }
```

Reply:
Oh gotcha. Thanks for catching that.